docs/docs/responses.mdx

```diff
@@ -164,12 +164,35 @@ Each button in the list of `buttons` should have two keys:
 * `title`: The text displayed on the buttons that the user sees.
 * `payload`: The message sent from the user to the assistant when the button is clicked.
 
+If you would like the buttons to also pass entities to the assistant:
+
+```yaml-rasa {4-8} title="domain.yml"
+responses:
+  utter_greet:
+  - text: "Hey! Would you like to purchase motor or home insurance?"
+    buttons:
+    - title: "Motor insurance"
+      payload: '/inform{{"insurance":"motor"}}'
+    - title: "Home insurance"
+      payload: '/inform{{"insurance":"home"}}'
+```
+Passing multiple entities is also possible with:
+
+```
+'/intent_name{{"entity_type_1":"entity_value_1", "entity_type_2": "entity_value_2"}}'
+```
+
 :::note bypass nlu with buttons
-It's common to use buttons as a shortcut to bypass the machine-learning-based NLU interpreter.
-Messages starting with `/` are sent straight to the
+It's common to use buttons as a shortcut to bypass the machine-learning-based NLU interpreter
+
+When interacting with the assistant through the shell or Rasa X, messages starting with `/` are sent straight to the
 `RegexInterpreter`, which expects NLU input in a shortened `/intent{entities}` format.
 In the example above, if the user clicks a button, the user input
 will be directly classified as either the `mood_great` or `mood_sad` intent.
+
+`entities` is a comma separated key-value pair of entity type and values to pass along with the intent.
+For example, `/inform{"ORG":"Rasa", "GPE":"Germany"}` will pass the intent `inform` and the entities `Rasa`
+and `Germany` which are of type `ORG` and `GPE` respectively to the `RegexInterpreter`. 
```

Comment:
Can this be rephrased as instruction instead of description? e.g. `You can pass entities directly to the RegexInterpreter in the following format...`

Reply:
done